west.yml

```diff
@@ -0,0 +1,23 @@
+# The west manifest file for upstream Zephyr.
+#
+# You are free to create your own manifest files and put them in any
+# repository you want, to create your own custom Zephyr installations.
+# For example, you could create a manifest file in your own out of
+# tree application directory, which would pull this zephyr repository
+# in as an ordinary project.
+
+west:
+  url: https://github.com/zephyrproject-rtos/west
+  revision: master
+
+manifest:
+  defaults:
+    remote: upstream
+    revision: master
+
+  remotes:
+    - name: upstream
+      url-base: https://github.com/zephyrproject-rtos
+
+  self:
+    west-commands: scripts/west-commands.yml
```

Comment:
+1 on this. Let's default to path `zephyr/` on the upstream manifest. Otherwise we'd need to change the doc substantially as well in case users choose a different repository name.

Reply:
Done